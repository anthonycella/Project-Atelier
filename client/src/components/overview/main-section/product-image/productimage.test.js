import renderer from 'react-test-renderer';
import ProductImage from './productimage.jsx';

it('Product Image component renders to the page', () => {
  const selection = { thumbnail_url: 'https://wegotthiscovered.com/wp-content/uploads/2021/08/free-guy-dude-1200x900.jpg' };
  const colorScheme = { foreground: 'black' };
  const component = renderer.create(
    <ProductImage photos={[selection]} selection={selection} selectionIndex={0} colorScheme={colorScheme} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
