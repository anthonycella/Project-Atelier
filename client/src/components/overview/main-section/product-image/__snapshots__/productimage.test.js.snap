// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Image component renders to the page 1`] = `
<div
  className="sc-iqcoie buPdTd"
>
  <div
    className="sc-iBkjds dZHXve"
  >
    <div
      className="sc-eCYdqJ iYCoDj"
    >
      <div
        className="sc-hKMtZM fkVqZn"
      />
      <div
        className="sc-jSMfEi hRYmjB"
      />
    </div>
    <div
      className="sc-bczRLJ iHbVwd"
    />
    <div
      className="sc-bczRLJ iHbVwd"
    />
    <div
      className="sc-bczRLJ iHbVwd"
    />
    <div
      className="sc-bczRLJ iHbVwd"
    />
    <div
      className="sc-gKXOVf eJFCRD"
    >
      v
    </div>
  </div>
  <div
    className="sc-ftvSup kElViT"
  >
    &lt;-
  </div>
  <div
    className="sc-kDDrLX hLpef"
  >
    <div
      className="sc-papXJ cSwewl"
    >
      []
    </div>
    <div
      className="sc-jqUVSM dBFLuP"
    >
      -&gt;
    </div>
  </div>
</div>
`;
